queries/just/highlights.scm

```diff
@@ -0,0 +1,39 @@
+
+(assignment (NAME) @variable)
+(alias (NAME) @variable)
+(value (NAME) @variable)
+(parameter (NAME) @variable)
+(setting (NAME) @keyword)
+(setting "shell" @keyword)
```

Comment:
Why can't this `shell` be at the bottom with other keywords

Reply:
These are not my queries, but I took a loot at this today. Seems like the `set shell := ...` setting is parsed differently from other settings, because part of the value of this setting is used to set the default injection language for regular Just recipes.

Essentially, "shell" is not a keyword and should actually be captured by `(NAME)` on the line above, but isn't because it's parsed weirdly.